packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/components/SimpleDropdownTask.js

```diff
@@ -0,0 +1,147 @@
+import { Markdownz, pxToRem } from '@zooniverse/react-components'
+import { Box, Select, Text, TextInput } from 'grommet'
+import { Down } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import counterpart from 'counterpart'
+import en from './locales/en'
+counterpart.registerTranslations('en', en)
+
+const StyledText = styled(Text)`
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+`
+
+// The 'other' option, aka 'allow user to create any answer' option, is
+// artificially disabled. Please refer to the README. (@shaunanoordin 20200820)
+const ENABLE_OTHER_OPTION = false
+
+function SimpleDropdownTask (props) {
+  const {
+    annotation,
+    className,
+    disabled,
+    task,
+  } = props
+  const { value } = annotation
+  
+  // Decide what kind of options to display.
+  // Use an array of objects instead of an array of text.
+  // This solves issues of duplicate text.
+  const optionsToDisplay = task.options.map(opt => ({
+    text: opt,
+  }))
+  
+  // If the Other option is enabled, we allow users to type in any text.
+  const otherOption = {
+    text: counterpart('Dropdown.otherLabel')
+  }
+  if (task.allowCreate && ENABLE_OTHER_OPTION) {
+    optionsToDisplay.push(otherOption)
+  }
+  
+  // Determine which option is selected.
+  const selectedOption = (value?.option)
+    ? optionsToDisplay[value?.selection]
+    : (value?.option === false)  // distinguishes between value simply not existing
+    ? otherOption
+    : undefined
```

Comment:
These ternary operators are really hard to follow. Maybe split this up into multiple lines so that it's easier to read.